cpp/modmesh/view/RApplication.cpp

```diff
@@ -43,6 +43,27 @@ RApplication::RApplication(int & argc, char ** argv)
     python::Interpreter::instance().preload_modules({"_modmesh_view", "modmesh"});
     pybind11::exec("modmesh.view = _modmesh_view");
 
+    RMenuBar * menuBar = new RMenuBar();
+    RMenu * fileMmenu = new RMenu(QString("File"));
+    RMenu * newMenu = new RMenu(QString("New"));
+    RAction * newFileAction = new RAction(
+        QString("New file"),
+        QString("Create new file"),
+        []()
+        {
+            qDebug() << "Create new file!";
+        });
+    RAction * exitAction = new RAction(
+        QString("Exit"),
```

Comment:
Interestingly, on mac, when this menu item string is `Exit`, the item does not show.  It shows after the string changes to something else.  I will study why it happens and fix it later.

Reply:
To be clear, the case we discovered should be the menu item string **starts with `Exit`**, not sure the contains `Exit` will also fail or not.